benchmarks/run_bayesmark.py

```diff
@@ -49,71 +47,76 @@ def run_benchmark(args: argparse.Namespace) -> None:
     cmd = (
         f"bayesmark-launch -n {args.budget} -r {args.repeat} "
         f"-dir runs -b {_DB} "
-        f"-o RandomSearch {samplers} "
+        f"-o {samplers} "
         f"-c {args.model} -d {args.dataset} "
         f"-m {metric} --opt-root benchmarks/bayesmark"
     )
     subprocess.run(cmd, shell=True)
 
-    cmd = f"bayesmark-agg -dir runs -b {_DB}"
-    subprocess.run(cmd, shell=True)
-
-    cmd = f"bayesmark-anal -dir runs -b {_DB}"
-    subprocess.run(cmd, shell=True)
-
 
 def make_plots(args: argparse.Namespace) -> None:
 
-    # https://github.com/uber/bayesmark/blob/master/notebooks/plot_test_case.ipynb
-    db_root = os.path.abspath("runs")
-    summary, _ = XRSerializer.load_derived(db_root, db=_DB, key=cc.PERF_RESULTS)
-    plot_warmup = json.loads(args.plot_warmup)
+    filename = f"{args.dataset}-{args.model}-partial-report.json"
+    df = pd.read_json(os.path.join("partial", filename))
+    summaries = (
+        df.groupby(["opt", "iter"])
+        .generalization.agg(["mean", "std"])
+        .rename(columns={"mean": "best_mean", "std": "best_std"})
+        .reset_index()
+    )
 
-    fig = plt.figure(figsize=(18, 8))
-    gs = fig.add_gridspec(1, 2)
-    axs = gs.subplots()
+    fig, ax = plt.subplots()
+    fig.set_size_inches(12, 8)
+    warmup = json.loads(args.plot_warmup)
+    metric = df.metric[0]
+    color_lookup = build_color_dict(df["opt"].unique())
 
-    for benchmark in summary.coords["function"].values:
-        for metric, ax in zip(["mean", "median"], axs):
-            make_plot(summary, ax, benchmark, metric, plot_warmup)
+    for optimizer, summary in summaries.groupby("opt"):
```

Comment:
It seems that the line color of each sampler could vary plot by plot. If it's possible, it would be better to keep using the same color for each sampler I think.
![optuna-breast-linear-sumamry](https://user-images.githubusercontent.com/181413/172030952-303eb4da-6dd5-44a9-9f82-5034da3bc393.png)
![optuna-diabetes-DT-sumamry](https://user-images.githubusercontent.com/181413/172030953-fbb78234-0669-4121-aeee-9e9036054d40.png)
(images from https://github.com/xadrianzetx/optuna-bayesmark/actions/runs/2379168197)

Reply:
Do you think we should produce color lookup table for each optimizer/pruner combo, or is it enough to make colors consistent throughoutt single benchmark run for subset of tested optimizers (so essentially making sure `build_color_dict(df["opt"].unique())` works on sorted values)?

Reply:
>  is it enough to make colors consistent throughout single benchmark run

Yes. I think this is a reasonable solution and consistent with the kurobako behavior.

Reply:
Implemented in d81b200bdbbcfe7528e1bae766fefe5f4d59be2c.